model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -53,6 +53,25 @@ def extract_holder_weights(constant_name, node_target, model, weights, to_numpy)
     return weights
 
 
+def get_node_args_spec(node: Node) -> List[str]:
+    """
+    Return a list of the node's function args & kwargs, so we can insert the kwarg as arg
+    Args:
+        node: Torch.fx.Node to extract target from, which the function the Node executes.
+
+    Returns: A list of args and kwargs names.
+
+    """
+    if node.target in [torch.nn.functional.conv1d, torch.nn.functional.conv2d, torch.nn.functional.conv3d]:
```

Comment:
why the special treatment for these specific layer types?

Reply:
Because `inspect.getfullargspec(node.target)` fails for them.

Reply:
only used here, so no need to add code for it.